src/services/llms/base.py

```diff
@@ -58,22 +58,26 @@ def _clean_response_text(self, text: str) -> str:
 
     def _parse_response(self, response_text: str) -> List[Dict[str, str]]:
         """
-        Parse the cleaned response text into structured data.
+        Parse the cleaned response text into structured review comments.
         
```

Comment:
Việc thay đổi mô tả hàm từ `Parse the cleaned response text into structured data` thành `Parse the cleaned response text into structured review comments` là không cần thiết và có thể gây hiểu nhầm. Mô tả ban đầu đã đủ rõ ràng.  Hãy sử dụng mô tả hàm tổng quát hơn, ví dụ: `Parse the response text into a structured format`.